src/taipy/gui/gui.py

```diff
@@ -1499,15 +1497,14 @@ def run(
                 If set to _True_, the Web server is run is a separated thread.
                 Note that if you are running in an IPython notebook context, the Web
                 server is always run in a separate thread.
-            async_mode (Optional[str]): The asynchronous model to use for the Flask-SocketIO.
+            async_mode (str): The asynchronous model to use for the Flask-SocketIO.
                 Valid values are:</br>
 
                 - "threading": Use the Flask Development Server. This allows the application to use
                   the Flask reloader (the *use_reloader* option) and Debug mode (the *debug* option).
                 - "eventlet": Use eventlet server.
                 - "gevent": Use gevent server.
-                - "gevent_uwsgi": Use uwsgi server.
-
+                </br>The default value is "threading"</br>
                 If this argument is not set, Taipy uses, in that order: `"eventlet"`, `"gevent_uwsgi"`,
```

Comment:
this line needs to be changed too ?

Reply:
yes imo. gevent and eventlet are what will mainly be used